src/leaflike/db.clj

```diff
@@ -0,0 +1,4 @@
+(ns leaflike.db
+  (:require [clojure.java.jdbc :as jdbc]))
+
+(defn db-spec [] {:connection-uri "jdbc:postgresql://localhost:5432/leaflike"})
```

Comment:
This is more of a discussion: If your function is returning the same result for every call does it make sense for it to be a value instead?